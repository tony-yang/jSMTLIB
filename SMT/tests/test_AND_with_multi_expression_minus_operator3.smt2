(set-logic QF_LRA)
(set-info :source | SMT-COMP'06 organizers |)
(set-info :smt-lib-version 2.0)
(set-info :category "check")
(set-info :status sat)
(set-info :notes |This benchmark is designed to check if the DP supports bignumbers.|)
(declare-fun x1 () Real)
(declare-fun x2 () Real)
(declare-fun x3 () Real)
(declare-fun x4 () Real)
(declare-fun x5 () Real)
(assert (and (> (- (* (- 6) x1) (* (- 3) x2)) 10) (> (- x3 (* (- 3) x3)) 20) (> (- (- x4) (- x5)) 30)))
(check-sat)
(exit)
