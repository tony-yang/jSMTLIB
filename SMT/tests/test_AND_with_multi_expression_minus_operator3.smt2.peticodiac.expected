# set_logic: QF_LRA
# :smt-lib-version: 2.0
# :category: "check"
# :status: sat
# :notes: |This benchmark is designed to check if the DP supports bignumbers.|
p cnf 5 3
c -6.0 3.0 0.0 0.0 0.0
b 5 >:10.0 NO_BOUND
c 0.0 0.0 4.0 0.0 0.0
b 6 >:20.0 NO_BOUND
c 0.0 0.0 0.0 -1.0 1.0
b 7 >:30.0 NO_BOUND
eoa
